net-nds/389-ds-base/389-ds-base-1.4.2.15.ebuild

```diff
@@ -69,15 +62,16 @@ src_configure() {
 		$(use_enable dna) \
 		$(use_enable bitwise) \
 		$(use_enable presence) \
-		$(use_with kerberos) \
 		$(use_enable auto-dn-suffix) \
+		$(use_with systemd) \
+		--with-systemdsystemunitdir="$(systemd_get_systemunitdir)" \
 		--with-initddir=no \
 		--enable-maintainer-mode \
```

Comment:
maintainer-mode is not used anymore.

Reply:
Thank you for those - unfortunately life has got in the way of working on this - I aim to be able to get back to it in October!

Reply:
you ok with forking this over to a PR of mine then? i could add you to metadata too of course, if you wanted.

Reply:
> 
> 
> you ok with forking this over to a PR of mine then? i could add you to metadata too of course, if you wanted.

Absolutely feel free! - happy to be added to metadata if you'd like someone else to help with maintaining :)